deploy/deploy-hyperlane.md

```diff
@@ -178,35 +177,176 @@ $  head -n 22 artifacts/agent_config.json
+<figure><img src="../.gitbook/assets/kurtosis-get-excited.png" alt=""><figcaption><p>Kurtosis will provision a brand new instance for you if this is your first time using Kurtosis.</p></figcaption></figure>
 
-Remember to set `--relayChains` or `HYP_BASE_RELAYCHAINS` appropriately.
+Once complete, you will be prompted to interact with your new, empty cloud instance via the Enclave Manager or via the Kurtosis CLI. Go ahead and click on Launch in Enclave Manager (see screenshot below). 
 
-Include the agent config from the [#2.-deploy-contracts](deploy-hyperlane.md#2.-deploy-contracts "mention") step in `CONFIG_FILES`. If you're using Docker, you will need to mount the file into the container.
+<figure><img src="../.gitbook/assets/kurtosis-launch-instance.png" alt=""><figcaption><p>Launch the Enclave Manager to interact with your new Cloud instance.</p></figcaption></figure>
 
-You should have already set up your relayer keys in step [#1.-setup-keys](deploy-hyperlane.md#1.-setup-keys "mention"), so you can skip that part of the [relayers](../operators/relayers/ "mention") guide.
+### Create the environment for your relayer and validator
+Next, you'll see a page where you can select to view existing enclaves or to create a new one. An enclave can be thought of as an isolated environment with which you can deploy things on to. In this case, it will be the environment that houses your relayer and validator. Select "Create Enclave".
 
-{% content-ref url="../operators/relayers/" %}
-[relayers](../operators/relayers/)
-{% endcontent-ref %}
+<figure><img src="../.gitbook/assets/kurtosis-create-enclave.png" alt=""><figcaption><p>Create a new enclave to house your relayer and validator.</p></figcaption></figure>
+
+The next screen will be the Kurtosis Package catalog: a collection of runnable packages that anyone can write and use. Start typing "hyperlane" in the search box and you will see the Hyperlane package appear, with the format: `github.com/kurtosis-tech/hyperlane-package`. Go ahead and select that and choose "Configure" in the bottom of your screen.
+
+<figure><img src="../.gitbook/assets/kurtosis-hyperlane-package-select.png" alt=""><figcaption><p>Select the Hyperlane package to configure and deploy</p></figcaption></figure>
+
+### Configure and deploy your relayer and validator
+You're almost done! Its now time to configure your relayer and validator. 
+
+In this guide, you will check the `Restart services` box, fill in only the required fields, and will an example configuration value. When deploying Hyperlane for your own project, please use your own configuration values and any of the optional fields as you see fit. For this example, the following `agent_config.json` file will be used:
+
+TODO: Add Sepolia to agent_config.json
+
+<details>
+
+<summary>Here is the example `agent_config.json` used in this section:</summary>
+
+```json
```

Comment:
I think a full config here may be overwhelming since they're so long. Also why does the config mix anvil and real testnets?

Reply:
Hey @jmrossy - spoke to Nam and Avi about this too and we decided that they would take the lead on revamping the docs and including Kurtosis in the new version. 

Original rationale: earlier parts of the guide include examples of deploying contracts to local Anvil network and setting up communications with real testnets like Goerli and Sepolia ([source](https://docs.hyperlane.xyz/docs/deploy/deploy-hyperlane#deploy)). To stay consistent, I opted to include those same networks. 

I am okay with removing the full configuration file and agree that it is overwhelming.